packages/combo-box/src/vaadin-combo-box-scroller-mixin.js

```diff
@@ -138,24 +138,21 @@ export const ComboBoxScrollerMixin = (superClass) =>
       this.addEventListener('click', (e) => e.stopPropagation());
 
       this.__patchWheelOverScrolling();
-
-      this.__virtualizer = new Virtualizer({
```

Comment:
Grid also initializes Virtualizer in `ready()`. Does it not have the same issue?

Reply:
I suppose it could be affected but the [test case](https://github.com/vaadin/web-components/blob/main/integration/tests/component-relayout-page.test.js) passes when adding an empty grid.